src/libraries/System.Runtime/ref/System.Runtime.cs

```diff
@@ -2899,6 +2899,8 @@ public sealed partial class ObsoleteAttribute : System.Attribute
         public ObsoleteAttribute(string? message, bool error) { }
         public bool IsError { get { throw null; } }
         public string? Message { get { throw null; } }
+        public string? DiagnosticId { get { throw null; } set { } }
```

Comment:
Please keep the same sorting that the generator (which we run periodically) will use. Eg., Message after DiagnosticId.